### PR TITLE
English editing for ffmpeg.markdown

### PR DESCRIPTION
**Pull request by AlucardZero**

### CODE REVIEW

File: source/_components/ffmpeg.markdown [hunk 1]
```diff
@@ -11,10 +11,10 @@ logo: ffmpeg.png
 ha_category: Hub
 ---
 
-It allow other Home-Assistant components to process video/audio streams. It need a ffmpeg binary in your system path. It support all ffmpeg version since 3.0.0. If you have a older version, please update.
+The FFmpeg component allows other Home Assistant components to process video and audio streams. This component supports all FFmpeg versions since 3.0.0; if you have a older version, please update.
 
 <p class='note'>
-You need a `ffmpeg` binary in your system path. On Debain 8 or Raspbian (Jessie) you can install it from backports. If you want Hardware support on a Raspberry Pi you need to build from source by yourself. Windows binary are avilable on the [FFmpeg](http://www.ffmpeg.org/) website.
+You need an `ffmpeg` binary in your system path. On Debian 8 or Raspbian (Jessie) you can install it from [debian-backports](https://backports.debian.org/Instructions/). If you want hardware acceleration support on a Raspberry Pi, you will need to build from source by yourself. Windows binaries are avilable on the [FFmpeg](http://www.ffmpeg.org/) website.
```
Comment (robbiet480) [inline source/_components/ffmpeg.markdown]: "a" not "an" :)

File: source/_components/ffmpeg.markdown [hunk 2]
```diff
@@ -27,20 +27,20 @@ ffmpeg:
...
 Configuration variables:
 
-- **ffmpeg_bin** (*Optional*): Default 'ffmpeg'. Set the ffmpeg binary.
+- **ffmpeg_bin** (*Optional*): Default 'ffmpeg'. The name or path to the `ffmpeg` binary.
 - **run_test** (*Optional*): Default True. Check if `input` is usable by ffmpeg.
 
 ### {% linkable_title Troubleshooting %}
 
-In most of case, `ffmpeg` autodetect all needed options to read a video/audio stream or file. But it is possible in rare cases that's needed to set a option to help `ffmpeg`. Per default `ffmpeg` use 5 seconds to detect all options or abort.
+In most cases, `ffmpeg` automatically detects all needed options to read a video or audio stream or file. But it is possible in rare cases that you will need to set an option to help `ffmpeg` out.
```
Comment (robbiet480) [inline source/_components/ffmpeg.markdown]: How about "you will need to set options to help `ffmpeg` out" instead?

File: source/_components/ffmpeg.markdown (hunk 1 above)
Comment (AlucardZero) [inline source/_components/ffmpeg.markdown]: I thought it was "eff eff em peg", so *an* is appropriate.  Or "the".

Comment (robbiet480) [inline source/_components/ffmpeg.markdown]: Let's go with "the". Better than either option.

**Comment by robbiet480** [comment]
Thanks! 🐬 🍪 💯

**Pull request merged**